Readme.md

```diff
@@ -9,26 +9,53 @@ This plugin is designed for [Prisma 2](https://www.prisma.io/blog/announcing-pri
 - Syntax highlighting
 - Auto-formatting
 - Linting
+- Jump-to-definition 
 - Auto-completion (_coming soon_)
-- Jump-to-definition (_coming soon_)
+
+It also includes an End-to-End test.
 
 ## Install
 
-Get the Prisma Extension from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=Prisma.prisma).
+- Get the Prisma Extension from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=Prisma.prisma).
 
 ## Preview
 
 ![](https://imgur.com/HbufPo6.png)
 
+## Structure
+
+```
+.
+--- client // Language Client
+|   --- src   
+|       --- extension.ts // Language Client entry point
+--- package.json // The extension manifest
+--- server // Language Server
+    --- src
+        --- server.ts // Language Server entry point
+```
+
 ## Development
 
-1. Run `yarn watch`
-2. Open this repository in vscode
-3. Press F5, a new file should open
-4. Change the language to Prisma
-5. Make a change to the syntax
-6. To reload, press the reload button in VSCode
-   1. **Developer: Inspect TM Scopes** is helpful for debugging syntax issues
+- Run `npm install` in this folder. This installs all necessary npm modules in both the client and server folder.
+- Run `yarn watch`.
+- Open VS Code on this folder.
+- Press `Ctrl+Shift+B` to compile the client and the server.
```

Comment:
I'm a bit confused about this setup. Are we compiling typescript inside this launch config?  

I might be misunderstanding, but I think I'd prefer if we built this in a non-VSCode specific way.

Something like this:

```sh
./node_modules/.bin/tsc       # compile my source to JS
./dist/bin/ls.js --port 6009  # start the language server on port 6009                     
```

Reply:
when running `yarn watch`, `tsc -b -w` is actually called. So the step mentioned in the Readmd.md is not needed, I'll remove that step